test/unit/spec/v2/view-builder/views/Identify_spec.js

```diff
@@ -0,0 +1,88 @@
+import IdentifierView from 'v2/view-builder/views/IdentifierView';
+import AppState from 'v2/models/AppState';
+import $sandbox from 'sandbox';
+import Settings from 'models/Settings';
+import IdentifyWithPasswordResponse from '../../../../../../playground/mocks/data/idp/idx/identify-with-password.json';
+
+describe('Identifier View shows custom links configured via config on load', function () {
+  beforeEach(function () {
+    this.init = (response, formName, View) => {
+      const currentViewState = {
```

Comment:
Just a bit tedious to manually create all config.
How about
- use `v2/MockUtil` to mock api response
- add another helper to create `appState` (run through transformers, and invoke `setIonResponse`. Basically mimic what BaseLoginRouter does)
- Then init `FormController`

Reply:
I didnt really see a pattern for this, so I followed what we have in EnrollWebauthn_spec. Let me see if using `v2/MockUtil` is any easier.

Reply:
Let's see if it's easier and we can refactor `EnrollWebauthn_spec` later on. cc @magizh-okta ^

Reply:
going to delete this test. Refactored to use e2e tests